### PR TITLE
LPS-153881 Create storage type field into New custom object modal

### PR DESCRIPTION
**Pull request by guilhermedcamacho**
Related tickets:

https://issues.liferay.com/browse/LPS-153881
https://issues.liferay.com/browse/LPS-154214

### CODE REVIEW

**Comment by guilhermedcamacho** [comment]
ci:test:sf

**Comment by guilhermedcamacho** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
ci:reevaluate:1377401_2498

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -10594,6 +10594,7 @@ object-action-trigger[onAfterUpdate]=On After Update
 object-configuration-name=Object
 object-definition-data=Object Data
 object-definition-id=Object ID
+object-definition-storage-type-tooltip=Select if the data will be stored in Liferay (default) or in an external system only. Limitations are applied for object definitions with external data bases with external services.
```
Comment (guilhermedcamacho) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:10597]: Hey @alinedoleron , can you change the hint message to end with: **Limitations are applied for object definitions with external services.** ?

**Comment by guilhermedcamacho** [comment]
Sent directly to Brian [brianchandotcom@118167](https://github.com/brianchandotcom/liferay-portal/pull/118167)

**Pull request closed without merging**